uniffi_docs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "uniffi_docs"
+version = "0.23.0"
+edition = "2021"
+description = "uniffi_meta"
```

Comment:
I removed authors section from here.
In other crates it's: 
```toml
authors = ["Firefox Sync Team <sync-team@mozilla.com>"]
```
But I'm not "Firefox Sync Team".